tests/templates/test_subroutines/test_qubitization.py

```diff
@@ -222,11 +222,13 @@ def test_qnode_autograd(self):
         assert qml.math.shape(res) == (4,)
         assert np.allclose(res, self.exp_grad, atol=1e-5)
 
+    # TODO: Allow the following cases once their underlying issues are fixed:
+    #  (True, 50000): jax.jit on jax.grad does not work with AmplitudeEmbedding currently
+    #  (False, 50000): Since #5774, the decomposition of AmplitudeEmbedding triggered by
+    #                  param-shift includes a GlobalPhase always. GlobalPhase will only be
+    #                  param-shift-compatible again once #5620 is merged in.
```

Comment:
Maybe include them in the parametrization but `xfail` these special cases inside the test case.